docs/installation.md

```diff
@@ -37,7 +37,7 @@ page.
 1. Clone the repo and change into the `nginx-gateway-fabric` directory:
 
    ```shell
-   git clone https://github.com/nginxinc/nginx-gateway-fabric.git
+   git clone https://github.com/nginxinc/nginx-gateway-fabric.git --branch v1.0.0
```

Comment:
Shouldn't we update this to use the manifests published by the release?

Reply:
I can update that afterwards I guess, since they aren't published yet and I want to verify the command.